sushichef.py

```diff
@@ -50,8 +48,42 @@ class NewzBeatChef(YouTubeSushiChef):
         'CHANNEL_THUMBNAIL': CHANNEL_THUMBNAIL,
         'CHANNEL_DESCRIPTION': CHANNEL_DESCRIPTION,
     }
+    DATA_DIR = os.path.abspath('chefdata')
+    DOWNLOADS_DIR = os.path.join(DATA_DIR, 'downloads')
+    ARCHIVE_DIR = os.path.join(DOWNLOADS_DIR, 'archive_{}'.format(CONTENT_ARCHIVE_VERSION))
+
+    def get_video_ids(self):
+        return get_video_ids(NEWZ_BEAT_CHANNEL_ID)
+
+    def get_channel_metadata(self):
+        return {
+            'defaults': {
+                'license': licenses.AllRightsLicense("NewzBeat"),
+                'high_resolution': True
+            }
+        }
+
+
+def get_video_ids(channel_id):
+    youtube = build('youtube', 'v3', developerKey = 'AIzaSyB55y0HJENbbEBQBQzM-jbhdkW3A4V6PMs')
```

Comment:
@richard-dinh For things like keys, it's best to specify them using environment variables, and throw an error if they're not set.

Reply:
My mistake, did not remove the key here when I was testing. Usually I would use a .env for this. Updated to check if `GOOGLE_API_KEY` exists and throws error if it does not.